ts/Core/Chart/Chart.ts

```diff
@@ -4169,4 +4169,10 @@ export default Chart;
  * @type {boolean|undefined}
  */
 
+/**
+ * The dimensions the user can pan the chart.
+ *
+ * @typedef {"x"|"y"|"xy"} Highcharts.ChartPanningTypeValue
+ */
+
```

Comment:
You can reduced this in the other doclet to `@declare Highcharts.OptionsChartPanningTypeValue` and keep there @validvalue and @type.